Jenkinsfile

```diff
@@ -1,9 +1,47 @@
-stage ('Build') {
-  node {
-    sh 'npm config set registry http://registry.npmjs.org/'
-    checkout scm
-    sh "mvn versions:set -DnewVersion=${env.BRANCH_NAME} -DgenerateBackupPoms=false"
-    sh 'mvn clean package -DskipTests -U -Djetty.log.level=WARNING -Djetty.log.appender=STDOUT'
-    archiveArtifacts artifacts: 'grakn-dist/target/grakn-dist*.tar.gz'
+node {
```

Comment:
this is gone because I do not need to manually build grakn anymore - Jenkins handles it. Otherwise yes I removed ALL tests to get things running. The only thing being tested is that we can build and start the distribution. The next pr will add the tests incrementally